### ISSUE TITLE
[BITV] 9.4.1.2/2.1 - The "More apps" menu in the mobile (responsive) view has been implemented using the aria-haspopup="menu" attribute, which informs users that a menu is being opened.(10)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41888]
However, even though the menu items can be navigated using arrow keys, the corresponding ARIA roles role="menu" and role="menuitem" are completely missing. 

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v2-n1

9.4.1.2/3.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n1

9.4.1.2/4.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v4-n1

9.4.1.2/5.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v5-n1

9.4.1.2/6.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v6-n1

9.4.1.2/7.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v7-n1

9.4.1.2/8.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v8-n1

9.4.1.2/9.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v9-n1

9.4.1.2/10.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v10-n1

9.4.1.2/11.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v11-n1
